lib/cypress/api_measure_evaluator.rb

```diff
@@ -86,12 +86,12 @@ def run_vendor_tests(vendor_link, measures, product_name, skip_c1_test, bundle_i
       @patient_links_task_hash.each do |patient_links|
         calcuate_cat_3(patient_links[0].split('/')[2], bundle_id)
         upload_test_execution(extract_test_execution_link(patient_links[1], 'C1'), patient_links[0].split('/')[2], true) unless skip_c1_test
-        # upload_test_execution(extract_test_execution_link(patient_links[1], 'C2'), patient_links[0].split('/')[2], false, skip_c1_test)
+        upload_test_execution(extract_test_execution_link(patient_links[1], 'C2'), patient_links[0].split('/')[2], false, skip_c1_test)
       end
-      # sleep(4)
-      # download_filter_data
-      # calculate_filtered_cat3(bundle_id)
-      # upload_c4_test_executions
+      sleep(4)
```

Comment:
Would it be possible to verify the data is in the expected state here instead of sleeping?